pkg/kubernetes/configurations.go

```diff
@@ -6,49 +6,104 @@
 package kubernetes
 
 import (
+	"encoding/json"
 	"strconv"
 
 	"github.com/dapr/cli/pkg/age"
+	"github.com/dapr/cli/utils"
 	v1alpha1 "github.com/dapr/dapr/pkg/apis/configuration/v1alpha1"
+	"github.com/gocarina/gocsv"
+	"gopkg.in/yaml.v2"
 	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// ComponentsOutput represent a Dapr component.
-type ConfigurtionsOutput struct {
-	Name            string `csv:"Name"`
-	TracingEnabled  bool   `csv:"TRACING-ENABLED"`
-	MTLSEnabled     bool   `csv:"MTLS-ENABLED"`
-	WorkloadCertTTL string `csv:"MTLS-WORKLOAD-TTL"`
-	ClockSkew       string `csv:"MTLS-CLOCK-SKEW"`
-	Age             string `csv:"AGE"`
-	Created         string `csv:"CREATED"`
+type configurationsOutput struct {
+	Name           string `csv:"Name"`
+	TracingEnabled bool   `csv:"TRACING-ENABLED"`
+	MetricsEnabled bool   `csv:"METRICS-ENABLED"`
+	Age            string `csv:"AGE"`
+	Created        string `csv:"CREATED"`
 }
 
-// List outputs all Dapr configurations.
-func Configurations() ([]ConfigurtionsOutput, error) {
+type configurationDetailedOutput struct {
+	Name string      `json:"name" yaml:"name"`
+	Spec interface{} `json:"spec" yaml:"spec"`
+}
+
+// PrintConfigurations prints all Dapr configurations.
+func PrintConfigurations(name, outputFormat string) error {
 	client, err := DaprClient()
 	if err != nil {
-		return nil, err
+		return err
 	}
 
 	confs, err := client.ConfigurationV1alpha1().Configurations(meta_v1.NamespaceAll).List(meta_v1.ListOptions{})
 	if err != nil {
-		return nil, err
+		return err
 	}
 
-	co := []ConfigurtionsOutput{}
+	filtered := []v1alpha1.Configuration{}
+	filteredSpecs := []configurationDetailedOutput{}
 	for _, c := range confs.Items {
-		co = append(co, ConfigurtionsOutput{
-			TracingEnabled:  tracingEnabled(c.Spec.TracingSpec),
-			Name:            c.GetName(),
-			MTLSEnabled:     c.Spec.MTLSSpec.Enabled,
-			WorkloadCertTTL: c.Spec.MTLSSpec.WorkloadCertTTL,
-			ClockSkew:       c.Spec.MTLSSpec.AllowedClockSkew,
-			Created:         c.CreationTimestamp.Format("2006-01-02 15:04.05"),
-			Age:             age.GetAge(c.CreationTimestamp.Time),
+		confName := c.GetName()
+		if (confName != "daprsystem") && (name == "" || confName == name) {
```

Comment:
It took me a few rereadings to parse this logic. Would it be more clear if expressed like this?:
```golang
if (confName == "daprsystem") {
    continue
}

if (name == "" || name == confName) {
...
```

Reply:
Done.